cura/Settings/PerObjectContainerStack.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Optional
+
+from UM.Decorators import override
+from UM.Logger import Logger
+from UM.Settings.Interfaces import PropertyEvaluationContext
+from UM.Settings.ContainerStack import ContainerStack
+
+
+class PerObjectContainerStack(ContainerStack):
+
+    @override(ContainerStack)
+    def getProperty(self, key: str, property_name: str, context: Optional[PropertyEvaluationContext] = None) -> Any:
+        if context is None:
+            context = PropertyEvaluationContext()
+        context.pushContainer(self)
+
+        # this import has to be here otherwise there will be a circular import loop
+        from cura.CuraApplication import CuraApplication
+        global_stack = CuraApplication.getInstance().getGlobalContainerStack()
+
+        # Handle the "limit_to_extruder" property.
+        limit_to_extruder = super().getProperty(key, "limit_to_extruder", context)
+        if limit_to_extruder is not None:
+            limit_to_extruder = str(limit_to_extruder)
+
+        # if this stack has the limit_to_extruder "not overriden", use the original limit_to_extruder as the current
+        # limit_to_extruder, so the values retrieved will be from the perspective of the original limit_to_extruder
+        # stack.
+        if limit_to_extruder == "-1":
+            if 'original_limit_to_extruder' in context.context:
```

Comment:
Use double quotes for strings.

Reply:
Fixed in 37414e7bc7c85ee85ce47289a3268f1214bda2b3